include/mrsh_limit.h

```diff
@@ -0,0 +1,8 @@
+#ifndef MRSH_LIMIT_H
+#define MRSH_LIMIT_H
+
+#ifndef PATH_MAX
+#define PATH_MAX 4094
```

Comment:
We should `#include <limits.h>` before this

Reply:
So far limits is included in the .c files before mrsh_limit.h is included. Do you prefer to just include mrsh_limit.h in the .c files and have the limits.h included in mrsh_limit.h?

Reply:
Updated the PR. Let me know if that's better. Then I'll squash the commits if you want.

Reply:
Looks good. Out of curiosity, why 4094 instead of 4096?

>Then I'll squash the commits if you want.

Please do!

Reply:
Oh gosh! That was a typo :) I'll fix that typo and squash the commits.

Reply:
done!